pkgs/applications/misc/orca/default.nix

```diff
@@ -34,8 +34,8 @@ buildPythonApplication rec {
   ];
 
   propagatedBuildInputs = [
-    # TODO: re-add liblouis when it is fixed
-    pygobject3 pyatspi dbus-python pyxdg brltty speechd gst-python setproctitle
+    gobject-introspection  # Temporary fix, see https://github.com/NixOS/nixpkgs/issues/56943
```

Comment:
Any reason adding it to `propagatedBuildInputs` and not `buildInputs`?  (in theory setting `strictDeps = false;` should work, but I haven't had any luck with that)

Reply:
Yeah, we only need to propagate `pythonDependencies`. And I think disabling `strictDeps` is better than adding build time dependencies to `buildInputs` and pretending `strictDeps = true` works.

Reply:
I'm getting the same runtime error when having `strictDeps = false;` though, no idea why. Seeing the same issue with `guake`.

Reply:
Yay, I already managed to forgot it is not overridable from `buildPythonPackage` https://github.com/NixOS/nixpkgs/pull/59187

WE should probably fix that.

Reply:
Yeah, a ton of python packages have that same problem :/

Reply:
A [fix](https://github.com/NixOS/nixpkgs/pull/60178) allowing to use `strictDeps = false;` is now on master. Please rebase and try that.

Reply:
Niceeeee!